src/services/EdFi.DataManagementService.Api/Program.cs

```diff
@@ -6,6 +6,10 @@
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
+app.UsePathBase("/api");
+app.UseRouting();
+
 app.MapGet("/", () => "Data Management Service");
+app.MapGet("/ping", () => Results.Ok(new { StatusCode = 200, CurrentDateTime = DateTime.Now.ToString() }));
 
 app.Run();
```

Comment:
You can add a test for this change by using `Microsoft.AspNetCore.Mvc.Testing` . This is a good walkthrough: https://www.twilio.com/en-us/blog/test-aspnetcore-minimal-apis